src/Symfony/Component/Mailer/composer.json

```diff
@@ -25,15 +25,7 @@
         "symfony/service-contracts": "^1.1|^2"
     },
     "require-dev": {
-        "symfony/amazon-mailer": "^4.4|^5.0",
-        "symfony/google-mailer": "^4.4|^5.0",
-        "symfony/http-client-contracts": "^1.1|^2",
-        "symfony/mailjet-mailer": "^4.4|^5.0",
-        "symfony/mailgun-mailer": "^4.4|^5.0",
-        "symfony/mailchimp-mailer": "^4.4|^5.0",
-        "symfony/messenger": "^4.4|^5.0",
-        "symfony/postmark-mailer": "^4.4|^5.0",
-        "symfony/sendgrid-mailer": "^4.4|^5.0"
```

Comment:
IMHO this is not needed

Reply:
`symfony/messenger` is needed, see the failing tests.

Reply:
Indeed, I added it back